### PR TITLE
Updated Shopify DOC

### PR DESCRIPTION
**Pull request by aureate-labs-team**
### Related Issues

closes #

### Pull Request Checklist

- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues

- I tested manually my code and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme
- [ ] I have written test cases for my code

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

### CODE REVIEW

**Comment by atlantis-vsf** [comment]
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs  or, to disable this message, set --silence-fork-pr-errors

**Comment by atlantis-vsf** [comment]
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs  or, to disable this message, set --silence-fork-pr-errors

**Comment by atlantis-vsf** [comment]
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs  or, to disable this message, set --silence-fork-pr-errors

File: packages/core/docs/shopify/api-client-reference.md [hunk 1]
```diff
@@ -0,0 +1,13542 @@
+
```
Comment (igorwojciechowski) [inline packages/core/docs/shopify/api-client-reference.md:1]: Seems like link to this file is not added to the sidebar in `packages/core/docs/.vuepress/config.js`

File: packages/core/docs/shopify/configuration.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+# Configuration
```
Comment (igorwojciechowski) [inline packages/core/docs/shopify/configuration.md:1]: Seems like a link to this file is not added to the sidebar in `packages/core/docs/.vuepress/config.js`

Comment (aureate-labs-team) [reply packages/core/docs/shopify/configuration.md:1]: Link added :)

File: packages/core/docs/shopify/api-client-reference.md (hunk 1 above)
Comment (aureate-labs-team) [reply packages/core/docs/shopify/api-client-reference.md:1]: Links Updated. Thanks.

File: packages/core/docs/shopify/use-cart.md [hunk 3]
```diff
@@ -84,57 +94,35 @@ export type Cart = {
...
 export default {
-  setup() {
-    const { cart, loadCart } = useCart();
-    
-    const products = computed(() => cartGetters.getItems(cart.value));
-    const totals = computed(() => cartGetters.getTotals(cart.value));
-    const totalItems = computed(() => cartGetters.getTotalItems(cart.value));
-    const checkoutUrl = computed(() => cartGetters.getCheckoutUrl(cart.value));
-    
+  setup () {
```
[the remark this answers is not in the document]
Comment (aureate-labs-team) [reply packages/core/docs/shopify/use-cart.md:112]: Fixed

File: packages/core/docs/shopify/use-category.md [hunk 4]
```diff
@@ -23,42 +24,44 @@ export type Category = {
...
 ```javascript
-import { onSSR } from '@vue-storefront/core';
-import { useCategory } from '@vue-storefront/shopify';
+import { useCategory } from '@vue-storefront/commercetools';
+import { onSSR } from '@vue-storefront/core'
 
 export default {
-  setup() {
-    const { categories, search } = useCategory('categories');
+  setup () {
```
[the remark this answers is not in the document]
Comment (aureate-labs-team) [reply packages/core/docs/shopify/use-category.md:42]: Removed whitespace

File: packages/core/docs/shopify/use-category.md [hunk 5]
```diff
@@ -23,42 +24,44 @@ export type Category = {
...
 
 ## Examples
 
-Fetches all collections on the shop, not including products.
+Fetches all collections on the shop, including products.
 
 ```javascript
-import { onSSR } from '@vue-storefront/core';
-import { useCategory } from '@vue-storefront/shopify';
+import { useCategory } from '@vue-storefront/commercetools';
```
[the remark this answers is not in the document]
Comment (aureate-labs-team) [reply packages/core/docs/shopify/use-category.md:38]: Yes, it should be. Thanks for finding :+1:

**Comment by atlantis-vsf** [comment]
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs  or, to disable this message, set --silence-fork-pr-errors

**Comment by atlantis-vsf** [comment]
Atlantis commands can't be run on fork pull requests. To enable, set --allow-fork-prs  or, to disable this message, set --silence-fork-pr-errors

**Pull request closed without merging**